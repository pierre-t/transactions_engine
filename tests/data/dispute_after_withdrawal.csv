type,client,tx,amount
deposit,1,1,1.0
deposit,1,2,2.0
deposit,1,3,5.0
withdrawal,1,4,8.0
dispute,1,3,
chargeback,1,3,
deposit,2,10,10
withdrawal,2,11,8.0
dispute,2,10,
deposit,3,20,10
deposit,3,21,20
dispute,3,21,
withdrawal,3,22,5.0
dispute,3,20,
resolve,3,20,
chargeback,3,21,
deposit,4,30,10
deposit,4,31,20
dispute,4,31,
withdrawal,4,32,5.0
dispute,4,30,
resolve,4,31,
chargeback,4,30,