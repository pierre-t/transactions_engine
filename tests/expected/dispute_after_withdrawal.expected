client,available,held,total,locked
1,0,0,0,true
2,0,2,2,false
3,5,0,5,true
4,20,0,20,true